src/Controller/SearchController.php

```diff
@@ -85,6 +85,16 @@ public function __invoke(Request $request)
             $request->query->set('locale', $channel->getDefaultLocale()->getCode());
         }
 
+        if (!$request->query->has('sort')) {
+            if (null !== $request->get('taxonCode')) {
```

Comment:
There is not `has` method on request. Also, this request allows `taxonCode` to be either set as an attribute (route parameter at products in taxon route) and as query parameter (all products route).